README.md

```diff
@@ -72,86 +113,87 @@ Resources that would help you get a better understanding of Flyte.
 - FB AI@Scale [Making MLOps & DataOps a reality](https://www.facebook.com/atscaleevents/videos/ai-scale-flyte-making-mlops-and-dataops-a-reality/1047312585732459/)
 - [GAIC 2020](http://www.globalbigdataconference.com/seattle/global-artificial-intelligence-virtual-conference-122/speaker-details/ketan-umare-113746.html)
 
-## Blog Posts
+### Blog Posts
  1. [Introducing Flyte: A Cloud Native Machine Learning and Data Processing Platform](https://eng.lyft.com/introducing-flyte-cloud-native-machine-learning-and-data-processing-platform-fb2bb3046a59)
  2. [Building a Gateway to Flyte](https://eng.lyft.com/building-a-gateway-to-flyte-474b451b32c8)
 
-## Podcasts
+### Podcasts
 - TWIML&AI -  [Scalable and Maintainable ML Workflows at Lyft - Flyte](https://twimlai.com/twiml-talk-343-scalable-and-maintainable-workflows-at-lyft-with-flyte-w-haytham-abuelfutuh-and-ketan-umare/)
 - Software Engineering Daily - [Flyte: Lyft Data Processing Platform](https://softwareengineeringdaily.com/2020/03/12/flyte-lyft-data-processing-platform-with-allyson-gale-and-ketan-umare/)
-- MLOps Coffee session - [Flyte: an open-source tool for scalable, extensible , and portable workflows](https://anchor.fm/mlops/episodes/MLOps-Coffee-Sessions-12-Flyte-an-open-source-tool-for-scalable--extensible---and-portable-workflows-eksa5k)
-
-
-# Features
- - Used at Scale in production by 500+ users at Lyft with more than *900k* workflow executed a month and more than *30+* million container executions per month
- - Fast registration - from local to remote in one second.
- - Centralized Inventory of Tasks, Workflows and Executions
- - Single Task Execution support - Start executing a task and then convert it to a workflow
- - gRPC / REST interface to define and executes tasks and workflows
- - Type safe construction of pipelines, each task has an interface which is characterized by its input and outputs. Thus illegal construction of pipelines fails during declaration rather than at
-   runtime
- - Types that help in creating machine learning and data processing pipelines like - Blobs (images, arbitrary files), Directories, Schema (columnar structured data), collections, maps etc
+- MLOps Coffee session - [Flyte: an open-source tool for scalable, extensible, and portable workflows](https://anchor.fm/mlops/episodes/MLOps-Coffee-Sessions-12-Flyte-an-open-source-tool-for-scalable--extensible---and-portable-workflows-eksa5k)
+
+<html>
+<h2 id="features"> 
+  🔥 Features
```

Comment:
What do you think this ordering of the main sections?

- Introduction
- Current Deployments
- Quick Start
- Features
- Component Repos
- Community & Resources
- Top Contributors

The rationale is to immediately show users where it's being used, how they can start, and what the key features are, then progressively go deeper into details and how they can contribute

Reply:
I think we should also add a status of the platform - like production ready etc.
Reason: Some folks have asked me - when will you be 1.0.0? We should probably explain that we are production ready.
Another thing we should probably think is if we should just release a 1.0.0 at some point soon

Reply:
@cosmicBboy Would the Current Deployments section include Available Plugins, Current Usage and Production K8s Operators? 
Or would these be in other sections?

Reply:
@SandraGH5 -- I think it'd be better if we include those in the separate sections.

@cosmicBboy  -- How about moving swapping `Current Deployments` and `Quick Start` sections? People usually would want to estimate how easy it is to get Flyte up and running.

Here's the final TOC:

- Introduction
- Quick Start
- Current Deployments
- Features
- Available Plugins
- Component Repos
- Production K8s Operators
- Community & Resources
- Top Contributors

Let me know if this is okay to proceed with.

@kumare3 -- We're already including that in the tagline below the Flyte logo -- "Flyte is a **production-grade**, container-native, type-safe workflow and pipelines platform optimized for large scale processing and machine learning written in Golang". Will this suffice?